DigitalLearningSolutions.Web/Views/Supervisor/ReviewCompetencySelfAsessment.cshtml

```diff
@@ -1,293 +1,260 @@
+                <button class="nhsuk-button nhsuk-u-margin-bottom-0" type="submit">
+                    Submit
+                </button>
+            </form>
+        </div>
     </div>
-  </div>
 }
 else if (ViewContext.RouteData.Values["viewMode"].ToString() == "View" && Model.Competency.AssessmentQuestions.First().SelfAssessmentResultSupervisorVerificationId != null)
 {
-  <h2>Review Outcome</h2>
-  @if (Model.Competency.AssessmentQuestions.First().Verified == null)
-  {
-    <p>Awaiting review by @((bool)Model.Competency.AssessmentQuestions.First().UserIsVerifier ? "you" : "another supervisor") .</p>
-  }
-  else
-  {
-
-    string verifiedString = "";
-    <dl class="nhsuk-summary-list">
-
-      <div class="nhsuk-summary-list__row">
-        <dt class="nhsuk-summary-list__key">
-          Reviewed
-        </dt>
-        <dd class="nhsuk-summary-list__value">
-          @if (Model.Verified != null)
-          {
-            DateTime verified = (DateTime)Model.Verified;
-            verifiedString = verified.ToString("dd/MM/yyyy");
-          }
-          @Html.Raw(verifiedString)<text> </text>@Model.SupervisorName
-        </dd>
-      </div>
-
-      <div class="nhsuk-summary-list__row">
-        <dt class="nhsuk-summary-list__key">
-          Confirmed
-        </dt>
-        <dd class="nhsuk-summary-list__value">
-          @if (Model.SignedOff)
-          {
-            <span class="nhsuk-tag nhsuk-tag--green">Yes</span>
-          }
-          else
-          {
-            <span class="nhsuk-tag nhsuk-tag--red">No</span>
-          }
-        </dd>
-      </div>
-
-      @if (Model.SupervisorComments != null)
-      {
-        <div class="nhsuk-summary-list__row">
-          <dt class="nhsuk-summary-list__key">
-            @(Model.DelegateSelfAssessment.ReviewerCommentsLabel == null ? "Reviewer comments" : Model.DelegateSelfAssessment.ReviewerCommentsLabel.ToString())
-          </dt>
-          <dd class="nhsuk-summary-list__value">
-            @Html.Raw(Model.SupervisorComments)
-          </dd>
-
-        </div>
-      }
-
-    </dl>
-    @if ((bool)Model.Competency.AssessmentQuestions.First().UserIsVerifier)
+    <h2>Review Outcome</h2>
+    @if (Model.Competency.AssessmentQuestions.First().Verified == null)
     {
-      <a class="nhsuk-button nhsuk-button--secondary" role="button" asp-action="ReviewCompetencySelfAssessment" asp-route-supervisorDelegateId="@ViewContext.RouteData.Values["supervisorDelegateId"]" asp-route-viewMode="Review" asp-route-candidateAssessmentId="@ViewContext.RouteData.Values["candidateAssessmentId"]" asp-route-resultId="@ViewContext.RouteData.Values["resultId"]">Update</a>
+        <p>Awaiting review by @((bool)Model.Competency.AssessmentQuestions.First().UserIsVerifier ? "you" : "another supervisor") .</p>
+    }
+    else
+    {
+        <dl class="nhsuk-summary-list">
+            <partial name="_ReviewOutcome"  model="Model"/>
+            @if (Model.SupervisorComments != null)

```

Comment:
please remove this comments from here as it is already in the partial view. It is appearing twice right now.
![image](https://github.com/TechnologyEnhancedLearning/DLSV2/assets/126668828/44d65001-640f-446e-9d0c-6d7428887f2c)

Reply:
This is addressed.